src/Countries/Tanzania.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+use IntlCalendar;
+
+class Tanzania extends Country
+{
+    public function countryCode(): string
+    {
+        return 'tz';
+    }
+
+    /** @return array<string, string|CarbonImmutable> */
+    protected function allHolidays(int $year): array
+    {
+        return array_merge([
+            'New Year\'s Day' => '01-01',
+            'Labor Day' => '05-01',
+            'Saba Saba Day' => '07-07',
+            'Farmers Day (Nane Nane Day)' => '08-08',
+            'Christmas Day' => '12-25',
+            'Boxing Day' => '12-26'
+        ], $this->variableHolidays($year));
+    }
+
+    /** @return array<string, string|CarbonImmutable> */
+    protected function variableHolidays(int $year): array
+    {
+        $easter = CarbonImmutable::createFromTimestamp(easter_date($year))
+            ->setTimezone('Africa/Dar_es_Salaam');
+
+        // Eid al-Fitr is on the first day of the 10th month in the calendar
```

Comment:
I'm still a bit hesitant to merge PRs with Islamic holidays, as there still isn't a good implementation yet.

Reply:
Let me remove Islamic holidays until good implementation  is found.